nucypher/blockchain/eth/signers/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""
+ This file is part of nucypher.
+
+ nucypher is free software: you can redistribute it and/or modify
+ it under the terms of the GNU Affero General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ nucypher is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU Affero General Public License for more details.
+
+ You should have received a copy of the GNU Affero General Public License
+ along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
+"""
+
+from nucypher.blockchain.eth.signers.base import Signer
+from nucypher.blockchain.eth.signers.software import ClefSigner, KeystoreSigner
+from nucypher.blockchain.eth.signers.hardware import TrezorSigner
```

Comment:
Does this mean that the user will be unable to import any of the signer classes if `trezorlib` is not installed (which, as far as I understand, is planned to be an extra, but I don't see it anywhere in the `setup.py` or `requirements` at the moment)?

Reply:
At the moment I'm leaning towards leaving it included in the standard installation - Nothing wrong with supporting the trezor "out of the box" IMO.

Reply:
I don't think it adds much to catch the import error of a package included in the standard install - if it did - why not wrap *any* import statement in a try/except?  It will already raise an `ImportError`.